src/pages/home/Home.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import axios from 'axios';
+import './Home.css';
+
+function Home() {
+    const [postData, setPostData] = useState([]);
```

Comment:
Kijkend naar wat voor type `postData` uiteindelijk is, dan is dat geen array, maar een object. Ik zou daarom de default waarde `undefined` maken.
```suggestion
    const [postData, setPostData] = useState(undefined);
```